hack/tool/terraform/aws/commands/ha/outputs.tf

```diff
@@ -27,6 +37,7 @@ locals {
             api = {
               externalAddress = module.k0sinfra.loadbalancer_dns
             }
+            images = var.k0s_airgap_bundle != "" ? local.k0s_spec_images_airgap : null
```

Comment:
I wonder if it would be better to have the possibility to have a real overlay for the k0s configuration This is not really airgap specific, as one could just use that image setting to use a self-hosted registry or any other things.

What about having a generic thing that can be used to pass in k0s configuration?

```tf
spec = merge(
  var.k0s_config_spec,
  { api = { externalAddress = module.k0sinfra.loadbalancer_dns, }, },
)


```

Reply:
Right, that is the end goal, and would allow for us to makesome really obscure testable k0s cluster configurations.

As it stands now, all of the autopilot inttests as well as the external build don't need this level of customization, so I'm going to focus on this minimal functionality first.

Reply:
The "airgap" suffix in the config name just feels a bit off to me here.